### DISCUSSION TITLE
Where are you hosting umami? (slow response times on Vercel)

### DISCUSSION THREAD

**Discussion by rohandebsarkar** [umami-software/umami#1249] [Q&A]
I am hosting umami on [Vercel](https://vercel.com/) and the PostgreSQL database on [Supabase](https://supabase.com/).

The response times are very high. The POST method times are over 2 seconds on average.

What hosting services are you using? And what are the average response times?

**Comment by mikecao** [comment]
I'm using DigitalOcean where I have a $20 droplet that I installed PostgreSQL on. Works fine.

**Reply by miroslavpejic85** [reply]
I too on DO, droplet $5 with PostgreSQL. For my usage case all ok!

@mikecao Take a look also on [Hetzner cloud](https://www.hetzner.com/cloud) IMHO one of the best server-cloud provider with a very good price! 

PS: Thank you so much for Umami, like it!

**Comment by rohandebsarkar** [comment]
Never mind, the problem was clearly on my side.

In my case, the database was hosted in Germany, but my Vercel serverless functions were hosted in the USA. This caused the major delay, once I changed the Vercel serverless function location to Germany, the response times were much quicker.

Anyone else facing this issue, try changing the [Vercel serverless function deploy location in the Vercel settings](https://vercel.com/docs/concepts/functions/serverless-functions/regions).

**Reply by anthonykusuma** [reply]
A quick Googling brought me to this discussion. Thank you!
